docs/providers/aws/events/README.md

```diff
@@ -14,6 +14,43 @@ layout: Doc
 
 Welcome to the AWS Serverless Events Glossary.
 
-Please select a section on the left to get started.
+Please select a section on the left to get started, or read on for general
+information regarding Lambda Events in Serverless.
 
 If you have questions, join the [chat in gitter](https://gitter.im/serverless/serverless) or [post over on the forums](https://forum.serverless.com/)
+
+
+## Usage
+
+Serverless enables creation, and management, of Lambda event triggers
+via the `events` configuration key. This key takes a list of objects
+which describe the events you wish your function to be subscribed to.
+
+For example, to create a function which listens to an SNS topic called
+`dispatch`, and also runs every two hours:
+```
+functions:
+  dispatcher:
+    handler: dispatcher.dispatch
+    events:
+      - sns: dispatch
+      - schedule: rate(2 hours)
+```
+
+The events managed within the `events` list may be referenced using
```

Comment:
To be honest I'm a bit skeptical about everything documented from this line and below.

In the SNS example, my understanding is that a new SNS topic will be created -> the documentation explains how to reference that topic. But this is specific to SNS. Not all events automatically create a resource, and we are not "referencing the event", but rather referencing the SNS topic. This kind of documentation could exist, but it would be a better fit in the SNS documentation.

TL/DR: I would remove the text below from this page.

Reply:
Great feedback, and very much along the lines of what I was looking for. For context, the biggest reason I wanted to add this was because I found it difficult to find documentation regarding how generated resources are/may be referenced when created within the `events` key.

With that in mind, I'd like to update line 40 onward to ensure it's accurate and applicable to all events, though I'm obviously a bit unfamiliar with some of the details, so I very much appreciate the guidance.

Reply:
I wonder if this might not be covered already (with possibly room for improvement) here: https://www.serverless.com/framework/docs/providers/aws/guide/resources#aws-cloudformation-resource-reference what do you think?

Reply:
Oh, yes, a link to that might make sense. I'll update shortly.

Reply:
So it seems as though most of the documentation was already in place, just not where I was expecting to find it. I've updated the PR with some additional links, removed my duplicate documentation, and retained a minor blurb just to note that it _is_ possible to reference the created resources.